syntax = "proto3";

package api.chat.response;

message DeleteMember {
    message Result {
    }
    message Error {
        enum Code {
            CHAT_NOT_FOUND = 0;
            YOU_ARE_NOT_MEMBER = 1;
            INSUFFICIENT_RIGHTS = 2;
            USER_IS_NOT_MEMBER = 3;
        }
        Code code = 1;
    }
    optional Result result = 1;
    optional Error error = 2;
}
